config/default.yml

```diff
@@ -3352,7 +3352,9 @@ Style/LineEndConcatenation:
                  Use \ instead of + or << to concatenate two string literals at
                  line end.
   Enabled: true
+  SafeAutoCorrect: false
   VersionAdded: '0.18'
+  VersionChanged: '0.63'
```

Comment:
I'm sorry I'm late about merging 💦
Please update this to `'0.64'`,  regenerate the document (Run `rake generate_cops_documentation` command) and solve CHANGELOG.md with the latest master branch.